tests/test_data/test_pipelines/test_transform/test_transform.py

```diff
@@ -79,6 +79,28 @@ def test_resize():
     assert np.equal(results['img'], results['img2']).all()
     assert results['img_shape'] == (800, 1280, 3)
 
+    results_seg = {
+        'img': img,
+        'img_shape': img.shape,
+        'ori_shape': img.shape,
+        'seg1': copy.deepcopy(img),
+        'seg2': copy.deepcopy(img),
+        'seg_fields': ['seg1', 'seg2']
+    }
+    transform = dict(
+        type='Resize',
+        img_scale=(640, 400),
+        multiscale_mode='value',
+        keep_ratio=False)
+    resize_module = build_from_cfg(transform, PIPELINES)
+    results_seg = resize_module(results_seg)
+    assert results_seg['seg1'].shape == results_seg['seg2'].shape
+    assert results_seg['img_shape'] == (400, 640, 3)
+    assert results_seg['img_shape'] != results_seg['ori_shape']
+    assert results_seg['seg1'].shape == results_seg['img_shape']
+    assert np.equal(results_seg['seg1'], results_seg['seg2']).all()
+    assert np.equal(results_seg['seg2'], results_seg['gt_semantic_seg']).all()
```

Comment:
If we changed the name, this line can be deleted.

Reply:
I have changed the name, please re-check. Thanks

Reply:
@AronLin